### ISSUE TITLE
Insert image tag when attributes differ

### ISSUE THREAD

**Issue by jaredmoody** [zhuochun/md-writer#164]
Currently image width, height and align are editable, but ignored when entered:

![jaredmoody com 2016-07-21 14-03-28](https://cloud.githubusercontent.com/assets/27690/17038737/fcfb9486-4f4b-11e6-8a87-05197e6f2f3f.png)

Inserts:

`![Hello](/images/2016/07/hello.jpg)`

I think either those fields should not be editable, or it should insert an html image tag with those attributes, right?

**Comment by zhuochun** [comment]
By default, these attributes are not used in the image tag. You can change the image tag in config to use them, e.g. 

```
imageTag: "{% img {align} {src} {width} {height} '{alt}' %}"
```

Refer to https://github.com/zhuochun/md-writer/wiki/Settings

**Comment by jaredmoody** [comment]
Having those settings editable by default but not do anything makes it seem like something is broken because you would expect changing them to have some effect. 

Maybe those fields could be disabled unless imageTag is set in the config?

**Comment by zhuochun** [comment]
Yep. Could disable the text box when imageTag didn't include the attribute placeholder. Thanks for the suggestion.
